hazelcast-spring/src/main/java/com/hazelcast/spring/HazelcastClientBeanDefinitionParser.java

```diff
@@ -243,7 +244,26 @@ private void handleLoadBalancer(Node node) {
         }
 
         private void handleNearCache(Node node) {
-            createAndFillListedBean(node, NearCacheConfig.class, "name", nearCacheConfigMap, "name");
+            BeanDefinitionBuilder nearCacheConfigBuilder = createBeanBuilder(NearCacheConfig.class);
```

Comment:
The `createAndFillListedBean()` couldn't handle the `preloader` vs. `preloaderConfig` mismatch, so I added a custom parsing code here.